pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -384,20 +400,37 @@ func (r *ReconcileIBMBlockCSI) updateStatus(instance *ibmblockcsi.IBMBlockCSI, o
 	return nil
 }
 
-func (r *ReconcileIBMBlockCSI) restartControllerPod(name string, namespace string) error {
-	pod := &corev1.Pod{}
-	statefulSetName := oconfig.GetNameForResource(oconfig.CSIController, name)
-	controllerPodName := fmt.Sprintf("%s-0", statefulSetName)
-	err := r.client.Get(context.TODO(), types.NamespacedName{
-		Name:      controllerPodName,
-		Namespace: namespace,
-	}, pod)
-
-	if err != nil {
-		return err
+func (r *ReconcileIBMBlockCSI) isAllPodImagesSynced(controllerStatefulset *appsv1.StatefulSet, controllerPod *corev1.Pod) bool {
```

Comment:
I suggest renaming the function to `areAllPodImagesSynced`

Reply:
done